packages/webdriverio/src/utils/index.ts

```diff
@@ -317,6 +317,16 @@ export async function findDeepElement(
         (this as WebdriverIO.Element).elementId
     )
     const { using, value } = findStrategy(selector as string, this.isW3C, this.isMobile)
+
+    /**
+     * if we are using a relative xpath selector and we have a parent element
+     * we need to fall back to the regular WebDriver Classic command as BiDi
+     * does not support relative xpath selectors with a start node
+     */
+    if (using === 'xpath' && (value.startsWith('./') || value.startsWith('..')) && (this as WebdriverIO.Element).elementId) {
```

Comment:
As mentioned in the other PR, can we use [`startNodes`](https://www.w3.org/TR/webdriver-bidi/#command-browsingContext-locateNodes) to enable this?